community-plugins.json

```diff
@@ -5333,5 +5333,13 @@
         "author": "mvdkwast",
         "repo": "mvdkwast/obsidian-copy-as-html",
         "branch": "master"
+    },
+    {
+        "id": "make-md",
+        "name": "MAKE.md",
+        "description": "The ultimate experience for makers on Obsidian",
```

Comment:
The description is a searchable field for users browsing the plugin gallery, I think you might want to provide a better description so that users know what the plugin does and have it more easily discoverable.

Reply:
Thanks, is there a character limit? I have one that's 264 characters long.

Reply:
Gotcha, I lowered it to under 256 for now and committed the change.